pkg/k8shandler/elasticsearch.go

```diff
@@ -367,6 +367,17 @@ func DoSynchronizedFlush(clusterName, namespace string) (bool, error) {
 
 	curlESService(clusterName, namespace, payload)
 
+	failed := 0
+	if shards, ok := payload.ResponseBody["_shards"].(map[string]interface{}); ok {
+		if failedFload, ok := shards["failed"].(float64); ok {
+			failed = int(failedFload)
+		}
+	}
+
+	if payload.Error == nil && failed != 0 {
+		payload.Error = fmt.Errorf("Number of shards to be flushed before restart can proceed: %d", failed)
```

Comment:
Provide a message that better describes what is happening and who is doing it: `Flushing N before  proceed with restart`

Reply:
what about something like "Failed to flush %d shards in preparation for cluster restart. Will attempt again."